Dockerfile

```diff
@@ -20,9 +20,15 @@ RUN BPF_CFLAGS="" \
     && BPF_CFLAGS=$BPF_CFLAGS go generate -v ./cmd/...
 RUN CGO_ENABLED=0 go build -v -o bin/eupf ./cmd/
 
-FROM alpine:3.18.5 AS runtime
+FROM alpine:3.18 AS runtime
 LABEL org.opencontainers.image.source="https://github.com/edgecomllc/eupf"
 
+RUN apk update \
+    # workaround to fix CVE-2023-5363, CVE-2023-5678. Use alpine 3.18.5 when these CVEs will be fixed there
```

Comment:
This issue has already been fixed in main. Verify that you don't rewrite main code.